lib/blpapi-wrapper.ts

```diff
@@ -257,9 +257,8 @@ export class Session extends events.EventEmitter {
     private doRequest(uri: string,
                       requestName: string,
                       request: any,
```

Comment:
I'll advocate remove the `request` para since it becomes irrelevant with the `doRequest` method.